src/admin/templates/optimize.db.tpl

```diff
@@ -1,130 +1,136 @@
-				{lng p="error"}
-			{else}
-				<img align="absmiddle" src="{$tpldir}images/ok.png" width="16" height="16" alt="" border="0" />
-				{lng p="success"}
-			{/if}</td>
-		</tr>
-		{/foreach}
-		</table>
-		
-		<p align="right">
-			<input class="button" type="button" value=" {lng p="back"} " onclick="document.location.href='optimize.php?sid={$sid}';" />
-		</p>
-	{elseif $executeStruct}
-	<form action="optimize.php?do=repairStruct&sid={$sid}" method="post" onsubmit="spin(this)">
-		<table class="list">
-		<tr>
-			<th>&nbsp;</th>
-			<th>{lng p="table"}</th>
-			<th width="120">{lng p="exists"}</th>
-			<th width="120">{lng p="structstate"}</th>
-			<th width="120">{lng p="status"}</th>
-		</tr>
-		{foreach from=$result item=table}
-		{cycle values="td1,td2" name=class assign=class}
-		<tr class="{$class}">
-			<td width="20"><img src="{$tpldir}images/db_table.png" border="0" alt="" width="16" height="16" /></td>
-			<td>{$table.table}</td>
-			<td>{if $table.exists}{lng p="yes"}{else}{lng p="no"}{/if}</td>
-			<td>{$table.missing} / {$table.invalid}</td>
-			<td align="right">{if !$table.exists || $table.missing || $table.invalid}
-				<img align="absmiddle" src="{$tpldir}images/error.png" width="16" height="16" alt="" border="0" />
-				{lng p="error"}
-			{else}
-				<img align="absmiddle" src="{$tpldir}images/ok.png" width="16" height="16" alt="" border="0" />
-				{lng p="ok"}
-			{/if}</td>
-		</tr>
-		{/foreach}
-		</table>
-		
-		<p>
-			<div style="float:left;">
-				{if $repair}<img src="{$tpldir}images/warning.png" border="0" alt="" width="16" height="16" align="absmiddle" />
-				{lng p="dbwarn"}{/if}
-			</div>
-			<div style="float:right;">
-				<input class="button" type="button" value=" {lng p="back"} " onclick="document.location.href='optimize.php?sid={$sid}';" />
-				{if $repair}<input class="button" type="submit" value=" {lng p="repairstruct"} " />{/if}
-			</div>
-		</p>
-	</form>
-	{else}
-	<form action="optimize.php?sid={$sid}&do=execute" method="post" onsubmit="spin(this)">
-		<table>
-			<tr>
-				<td>{lng p="tables"}:</td>
-				<td>{lng p="action"}:</td>
-			</tr>
-			<tr>
-				<td valign="top"><select size="10" name="tables[]" multiple="multiple">
-				{foreach from=$tables item=table}
-					<option value="{$table}" selected="selected">{$table}</option>
-				{/foreach}
-				</select></td>
-				<td valign="top">
-					<table>
-						<tr>
-							<td valign="top" width="20" align="center"><input type="radio" id="op_optimize" name="operation" value="optimize" checked="checked" /></td>
-							<td valign="top"><img src="{$tpldir}images/db_optimize.png" border="0" alt="" width="32" height="32" /></td>
-							<td><label for="op_optimize"><b>{lng p="op_optimize"}</b></label><br />
-								{lng p="op_optimize_desc"}</td>
-						</tr>
-						<tr>
-							<td colspan="3">
-								&nbsp;
+
+	<div class="alert alert-warning">{lng p="dbwarn"}</div>
+
+	{if $execute}
```

Comment:
bitte isset oder empty wieder einsetzen.

Reply:
Comes with the next adjustments